_assignments/mp4.md

```diff
@@ -0,0 +1,192 @@
+
+---
+
+## Part 2 - Make it a Game!
+
+### Shooting Game
+
+You are now going to take your flight simulator, and turn it into a shooting game.
+
+1. Add a weapon, either a laser or physical bullets/rockets, to your plane. You decide from where the ray/bullets emanate from the plane. 
+   Do not use Unity’s Debug.DrawLine for your laser, as it will not appear in a built executable.
+
+2. Scatter some spheres in the sky. When your plane successfully shoots a sphere, the sphere should disappear for 5 seconds, and then 
+   reappear. Make sure the spheres are easily visible.
+
+3. Successfully shooting the spheres should increment the game score, which we will talk about in the next section.
+
+
+### UI
+
+You must implement a User Interface for your game. Remember to consult the Oculus Best Practices Guide and share ideas with your teammates. 
+If you are still confused, think about the games you have played before, and check [this](https://developer.oculus.com/blog/introducing-the-oculus-sample-framework-for-unity-5/).
+
+1. You should have a pause button that pauses the game, and provides the option to go back to the main menu.
+
+2. Your UI should show the score to players. After hitting a sphere in your scene, your score should increase.
+
+
+### Tutorial
+
+Since you came up with some fancy control scheme, the likes of which we may never have seen before, you must give us a tutorial. 
+You can assume we know how to use the pointing device to access the start menu, but nothing else.
+
+1. Your tutorial must cover all of the plane controls.
+
+2. The tutorial can be accessed from the main menu, or simply be in the same room as the start menu. 
+
+3. For 20 points of extra credit, make your tutorial interactive. This means the player should initiate the tutorial, and receive 
+   prompts to perform actions. Upon performing these actions, the player should receive feedback, and the tutorial should advance. 
+   For example, the tutorial could prompt the player to “Throw the controller across the room in order to accelerate”. Upon hurling the controller, 
+   the player would see some indication that the plane has sped up, and get a message saying “good job”.
+
+
+### Sound
+
+Add all necessary sounds to your simulator. Some examples include engine noise, crash sounds, firing sounds, ambient music, etc. Make sure that your 
+sounds are neither too loud, nor too quiet to hear, and make for a comfortable experience.
+
+
+### "Game Over" Condition
+
+When the player crashes the plane, they should see a game over screen, which should give the option to go back to the main menu, restart the game, or quit.
+If your game over screen has a menu, you can choose how it will be controlled, but make sure this is communicated to the player via the tutorial, or messages 
+included in the “game over” menu screen (such as “juggle the touch controllers to return to the main menu”).

```

Comment:
NO NO NO NO PLEASE DO NOT JUGGLE THE TOUCH CONTROLLERS!!!

Reply:
There are supposed to be footnotes for this and line 161...

Reply:
Is there a better way to do footnotes than how I did it in the last commit?